src/ppx/graphics_util.cpp

```diff
@@ -102,17 +102,28 @@ Result CopyBitmapToImage(
     // Scoped destroy
     grfx::ScopeDestroyer SCOPED_DESTROYER(pQueue->GetDevice());
 
-    // Row stride alignment to handle DX's requirement
-    uint32_t rowStrideAlignement = grfx::IsDx12(pQueue->GetDevice()->GetApi()) ? PPX_D3D12_TEXTURE_DATA_PITCH_ALIGNMENT : 1;
-    uint32_t alignedRowStride    = RoundUp<uint32_t>(pBitmap->GetRowStride(), rowStrideAlignement);
+    // This is the number of bytes we're going to copy per row.
+    uint32_t rowCopySize = pBitmap->GetWidth() * pBitmap->GetPixelStride();
+
+    // When copying from a buffer to a image/texture, D3D12 requires that the rows
+    // stored in the source buffer (aka staging buffer) are aligned to 256 bytes.
+    // Vulkan does not have this requirement. So for the staging buffer, we want
+    // to enforce the alignment for D3D12 but not for Vulkan.
+    //
+    uint32_t apiRowStrideAlignement = grfx::IsDx12(pQueue->GetDevice()->GetApi()) ? PPX_D3D12_TEXTURE_DATA_PITCH_ALIGNMENT : 1;
```

Comment:
typo `alignment` (also in the comment, line below)

Reply:
Thanks. Fixed.